packages/flutter/lib/src/material/app.dart

```diff
@@ -743,6 +743,14 @@ class MaterialScrollBehavior extends ScrollBehavior {
       case TargetPlatform.windows:
         return child;
       case TargetPlatform.android:
+        if (updatedAndroidOverscrollIndicator) {
+          return StretchingOverscrollIndicator(
+            axisDirection: details.direction,
+            child: child,
+          );
+        }
+        continue glow;
+      glow:
```

Comment:
do you need the `glow` label here? I though I simple `continue` at the end of a case also just continues executing the next case:

```suggestion
        continue;
```

Maybe I am confusing programming languages here, though.

Reply:
I thought so too, I expected it to just fall through, but the `continue` was necessary, which necessitated specifying a place to `continue` to.. I'll double check.

Reply:
Out of curiosity I just gave this a try in dartpad. Looks like dart requires a label after the continue in a switch case. Just ignore my comment here and sorry for the noise :)